.github/dependabot.yml

```diff
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: "pip"
```

Comment:
This part will basically mimic the tip targets in Jenkins by creating a PR with updated dependencies and the added benefit of:

2. If the build passes, we can straight merge those PRs.
3. If it does not, we see that some style needs to be fixed.

If we try and like this approach, then we could remove the tip jobs in Jenkins.

Reply:
This is false, because tip targets check against the last version of python.

Reply:
I like this approach, but I can't find documentation on which requirement files dependabot will update, at least not in [1]. Will it update `dev/requirements.txt` and `test-requirements.txt`? Do you happen to have a reference for this?

https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file

Reply:
AFAICT, at the moment one only can specify the directory to search the manifest files and `dependabot` will discover them there. That's why I put the file under `dev/` and configure it in line 4.

I found some discussions explaining that behavior as for example: https://github.com/dependabot/dependabot-core/issues/3940#issuecomment-1227005267

The source code is here: https://github.com/dependabot/dependabot-core/blob/main/python/helpers/lib/parser.py

Reply:
Found it, apparently it probes `**/.txt`:
```
    requirement_files = glob.glob(os.path.join(directory, '*.txt')) \
        + glob.glob(os.path.join(directory, '**', '*.txt'))
```
Thanks for digging it up!